criu/util.c

```diff
@@ -740,6 +740,7 @@ int is_empty_dir(int dirfd)
 
 	fdir = fdopendir(dirfd);
 	if (!fdir) {
+		 pr_perror("open directory stream by fd %d failed", dirfd);
```

Comment:
pls remove an extra space after two tabs.

Reply:
sorry for this mistake.. fixed